### PR TITLE
Aswathy/refactor: refactored to typescript for show,localized component

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_dhero-dmt5.tsx [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
-import PropTypes from 'prop-types'
 import { localize } from 'components/localization'
 import { Flex } from 'components/containers'
 import { QueryImage, Header } from 'components/elements'
 import { Button, LinkButton } from 'components/form'
 import device from 'themes/device'
 import useHandleSignup from 'components/hooks/use-handle-signup'
 
+type DHeroProps = {
+    background?: string | JSX.Element
```
Comment (NikitK-deriv) [inline src/components/custom/_dhero-dmt5.tsx:11]: Can we type it properly with `ImageDataLike | IGatsbyImageData` from gatsby-plugin-image?
Note: need to import just types like this: `import type { IGatsbyImageData, ImageDataLike } from 'gatsby-plugin-image'`

As far as I know `QueryImage` can get as background only these 2 types of properties

File: src/components/custom/_dhero-dmt5.tsx [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+    background_alt?: string | JSX.Element
+    background_image_name?: string
+    content?: string | JSX.Element
+    go_to_live_demo?: boolean
+    image_name?: string
+    is_mobile?: string | boolean
+    join_us_for_free?: boolean
+    Logo?: string | JSX.Element
+    title?: string
```
Comment (NikitK-deriv) [inline src/components/custom/_dhero-dmt5.tsx:20]: We can see in file by path `src/pages/dmt5/index.tsx` that we use only 6 of this properties. Please help to make necessary properties like title, content, etc.

**Review by NikitK-deriv — COMMENTED** [review]
Please resolve all ts errors in file 🙏

File: src/components/custom/_dhero-dmt5.tsx (hunk 1 above)
Comment (aswathy-deriv) [reply src/components/custom/_dhero-dmt5.tsx]: okay

**Review by michio-rework — APPROVED** [review]

**Review by michio-rework — APPROVED** [review]

**Review by michio-rework — CHANGES_REQUESTED** [review]
it seems there has been a change on `_dhero-dmt5.js`, please resolve the conflicts, thanks thanks :pray:

**Pull request closed without merging**